target/linux/mpc85xx/files/arch/powerpc/boot/dts/tl-wdr4900-v1.dts

```diff
@@ -271,6 +316,11 @@
 	leds {
 		compatible = "gpio-leds";
 
+		wps {
+			gpios = <&ath9k 1 GPIO_ACTIVE_LOW>;
+			label = "tp-link:blue:wps";
```

Comment:
nitpick: led is green

Reply:
Fixed.

Reply:
Although to be honest these should be using the new LED macros.